scheduler/core/components/ranker/default.py

```diff
@@ -29,6 +29,12 @@ def _default_score_combiner(x: npt.NDArray[float]) -> npt.NDArray[float]:
     # Note we need to use 0. or applying this function results in an array of int instead of float.
     return np.array([np.max(x)]) if 0 not in x else np.array([0.])
 
+
+# Default telescope altitude limits
+_def_alt_limits_site: Dict[Site, Dict] = {Site.GS: {'min': 18.0*u.deg, 'max': 88.0*u.deg},
+                                     Site.GN: {'min': 18.0*u.deg, 'max': 88.0*u.deg}}
+_def_alt_limits: Dict[str, float] = {'min': 18.0*u.deg, 'max': 88.0*u.deg}
+
```

Comment:
This is what we talked about. Instead of repeatedly using "magic" keywords of `'min'` and `'max'`, I would first create an `Enum` to represent these. This is probably best placed somewhere in lucupy since it is general and we could easily find cause to reuse this.

```python
from enum import auto, Enum

class MinMax(Enum):
    '''
    Enum typically to be used as dictionary keys when we have a dictionary
    that has both a min value and a max value key: that way we do not have
    to repeat the magic strings 'min' and 'max' in the code.

    Furthermore, it will always be the case that MIN < MAX, in case that is needed.
    '''
    MIN = auto()
    MAX = auto()
```

Then these would become:

```python
# This is just a tip to the IDE that these values should not be changed.
from typing import Dict, Final

from astropy import units as u
from astropy.coordinates import Angle
from lucupy.minimodel import Site

# If we don't wrap the angles in Angle, we get Quantity.
_def_alt_limits_site: Dict[Site, Dict[MinMax, Angle]] = {
    Site.GS: {MinMax.MIN: Angle(18.0 * u.deg), MinMax.MAX: Angle(88.0 * u.deg)},
    Site.GN: {MinMax.MIN: Angle(18.0 * u.deg), MinMax.MAX: Angle(88.0 * u.deg)}
}

# Here, you're not using a float as the value since you're multiplying by a unit.
# Again, this gives Quantity, but we probably want Angle.
_def_alt_limits: Dict[MinMax, Angle] = {
    MinMax.MIN: Angle(18.0 * u.deg),
    MinMax.MAX: Angle(88.0 * u.deg)
}